docs/reference/cli/mcpchecker_result_view.md

```
@@ -1,17 +1,17 @@
 ### Synopsis
 
-Render the JSON output produced by "mcpchecker run" in a human-friendly format.
+Render the JSON output produced by "mcpchecker check" in a human-friendly format.
 
 Examples:
-  mcpchecker view mcpchecker-netedge-selector-mismatch-out.json
-  mcpchecker view --task netedge-selector-mismatch --max-events 15 results.json
+  mcpchecker result view mcpchecker-netedge-selector-mismatch-out.json
+  mcpchecker result view --task netedge-selector-mismatch --max-events 15 results.json
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)